src/index.html

```diff
@@ -0,0 +1,212 @@
+                                    </p>
+                                </li>
+                    <li class="lang-skill">
+                                    <img src="../assets/html.png" alt="css">
+                                    <p>
+                                        HTML
+                                    </p>
+                                </li>
+                </div>
+                <li class="lang">
+                    <p>
+                        FrameWorks
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+                <li class="lang">
+                    <p>
+                        Skills
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+            </ul>
+        </div>
+    </section>
+    <section id="contact" class="contact-me">
+        <div class="bg-image">
+            <h2 class="contact-title">Contact me</h2>
+            <p class="contact-text">
+                If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I’d love to help
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here" required></textarea>
+                    <p class="alert"></p>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
+            </form>
+        </div>
+    </section>
+
+    <div class="modal-example">
+        <div class="modal-head">
+          <h2 class="projectname"></h2>
+          <button type="button" class="closeModal">&times;</button>
+        </div>
+        <div class="modal__body">
+          <div class="short_desc">
+            <h4 class="company"></h4>
+            <span></span>
+            <h4 class="role"></h4>
+            <span></span>
+            <h6 class="year"></h6>
+          </div>
+          <img src="#" alt="img" class="modal-img">
+          <div class="project-desc">
+            <p class="desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. In voluptatem pariatur modi rerum error
+              ea consequatur, excepturi eum maiores! Quo quibusdam culpa ratione.?Lorem ipsum dolor sit amet, consectetur
+              adipisicing elit.
+              !</p>
+            <div class="desc_btn">
```

Comment:
- [ ] Hello, great effort working on your project, but as requested in the project, you should create the pop up also dynamically and the hard coded line of code in your HTML file should be removed, just the way you have done for the project cards. Kindly remove it, only then we can be sure your pop up card is created dynamically. 👍 👍